h2o-genmodel/src/main/java/hex/genmodel/algos/rulefit/MojoCondition.java

```diff
@@ -0,0 +1,50 @@
+package hex.genmodel.algos.rulefit;
+
+import java.io.Serializable;
+
+public class MojoCondition implements Serializable {
+
+    public enum Type {Categorical, Numerical};
+    public enum Operator {LessThan, GreaterThanOrEqual, In}
+    int featureIndex;
+    Type type;
+    public Operator operator;
+    public String featureName;
+    public boolean NAsIncluded;
+    public String languageCondition;
+    public double numTreshold;
+    public String[] languageCatTreshold;
+    public int[] catTreshold;
```

Comment:
Naming with leading underscore? I know it's pain and against Java convention but if it is project conventions the it is the Law 🦊